droidlet/dashboard/web/src/StateManager.js

```diff
@@ -310,7 +329,16 @@ class StateManager {
     }
     this.memory.chats = res.allChats;
 
-    // once confirm that this chat has been sent, clear last chat action dict
+    // Set the commandState to display 'received' for one poll cycle and then switch
+    this.memory.commandState = "received";
+    setTimeout(() => {
+      if (this.memory.commandState === "received") {
+        // May have moved on already, in which case leave it
+        this.memory.commandState = "thinking";
+      }
+    }, this.memory.commandPollTime - 1); // avoid race condition
```

Comment:
Eventually we will need to query agent memory for chat status -- is it unprocessed or is already processed and determine `commandState` based on that. For now this one works.